### ISSUE TITLE
Merging pages is adding spaces in text

### ISSUE THREAD

**Issue by BeeFreeHere** [torakiki/pdfsam#300]
After merging 50+ files a lot, not everywhere, of text is altered and the output shows spaces between letters.

Here is a short view of what is wrong:
![pdfmerge bug](https://user-images.githubusercontent.com/42585481/44420314-e1a54580-a57d-11e8-8ba0-30f71af965f1.png)

Check page 23. It shows spaces in the merged result and I miss the last "8" of the text.
Versie 0.01.pdf

The original page 23.
[23.pdf](https://github.com/torakiki/pdfsam/files/2307487/23.pdf)

Maybe it has to do with the amount/size of files I add. In the produced example 12 files.
![pdf merge bug 2](https://user-images.githubusercontent.com/42585481/44420413-23ce8700-a57e-11e8-99e9-ec2d441f9377.PNG)

For what I see it seems to happen mostly with capital formatted Arial.
The maiden pdf files where produced with Excel and Word.

I purchased PDFMerge because previous work was very nice but this might force me to look further. I need the files merged pretty soon!

Hope you can fix this bug.

With kind regards,

Rudi Olthof

**Comment by torakiki** [comment]
Could you please attach the other files so that I can try to reproduce it? I tried some different merge tasks using 23.pdf and some other test PDFs but everything worked fine so I'd need the files and a screenshot of the settings to try to execute the same exact task you are performing.
Is there any information or error message in the Log section?

**Comment by BeeFreeHere** [comment]
Hi torakiki,

Sure, here we go :-)

[00.pdf](https://github.com/torakiki/pdfsam/files/2310755/00.pdf)
[01.pdf](https://github.com/torakiki/pdfsam/files/2310757/01.pdf)
[02 - 03.pdf](https://github.com/torakiki/pdfsam/files/2310758/02.-.03.pdf)
[04 - 11.pdf](https://github.com/torakiki/pdfsam/files/2310761/04.-.11.pdf)
[12.pdf](https://github.com/torakiki/pdfsam/files/2310762/12.pdf)
[13 - 14.pdf](https://github.com/torakiki/pdfsam/files/2310764/13.-.14.pdf)
[15.pdf](https://github.com/torakiki/pdfsam/files/2310765/15.pdf)
[16.pdf](https://github.com/torakiki/pdfsam/files/2310766/16.pdf)
[17 - 19.pdf](https://github.com/torakiki/pdfsam/files/2310767/17.-.19.pdf)
[20.pdf](https://github.com/torakiki/pdfsam/files/2310768/20.pdf)
[21 - 22.pdf](https://github.com/torakiki/pdfsam/files/2310770/21.-.22.pdf)
[23.pdf](https://github.com/torakiki/pdfsam/files/2310771/23.pdf)

If possible please remove these after you got them since the data in them is sensitive.

With kind regards,

Rudi Olthof

**Comment by torakiki** [comment]
Got them and deleted. Unfortunately I couldn't reproduce it, everything seems to merge fine but my file is 1.6mb bigger then yours.. could you add a screeshot of the merge settings? is there anything in the Log? Is this consistently happening if you repeat the merge?

**Comment by BeeFreeHere** [comment]
Hey Torakiki,

To get the screenshot of the settings I changed language from Dutch to English so it is easier to read for you.

Here are the settings:
![image](https://user-images.githubusercontent.com/42585481/44476091-0c070980-a637-11e8-8269-c213b3e61f56.png)
Btw, i haven't changed merge settings in any way.

This screenshot was made after I did another attempt to merge them with PDFSam. Note that some files are altered. I wasn't happy with the sharpness of some pictures.

No notification of errors, however here is the log telling me a new fonts was found and containing an error.
![image](https://user-images.githubusercontent.com/42585481/44476310-b1ba7880-a637-11e8-8fbd-2946e405dbff.png)
![image](https://user-images.githubusercontent.com/42585481/44476340-c26aee80-a637-11e8-96e7-d9c2eb85e905.png)
![image](https://user-images.githubusercontent.com/42585481/44476366-cf87dd80-a637-11e8-8deb-32a00b38c639.png)

Checking the result now, to my surprise it suddenly seems to work!
Previously i did testing with different (i tried all) output versions witch always resulted in the spaces bug.

I tried to make the whole book and again the result looks fine now.

As me being a software engineer myself I know how frustrating it is to not have found the cause of this. I am sorry for your time so far and will keep the pdf's in a backup next time i come across this  issue.
The once i send previously are also gone for me.

With kind regards,

Rudi
